src/services/userHistoryItem.tsx

```diff
@@ -157,16 +157,34 @@ export const getHistoryItemsIndexFromServer = async (bearerToken?: string) => {
   return generateHistoryItemsIndexDictionary(userHistoryItems)
 }
 
-/*
-  Mini HistoryItem object:
-  d = mediaFileDuration
-  p = userPlaybackPosition
-  c = completed
-  m = mediaRefId
-  e = episodeId
```

Comment:
These type of comment blocks are better as JSDoc strings because JSDoc strings will actually show up in your editor as you hover over the properties.

Reply:
👍